### ISSUE TITLE
Creating a new additional payment for an order raises ActionView::MissingTemplate error

### ISSUE THREAD

**Issue by robodisco** [komoju/spree_komoju#33]
Hi @rramsden - good to see you at the Kaigi,

In Spree when we add an additional item to an order we want to charge the customer the outstanding balance. Going to the orders payment tab in the admin and clicking "new payment" and they trying to save raises a missing template error.

Here's is the stack trace

```
Showing /Users/adam/.rbenv/versions/2.2.0/lib/ruby/gems/2.2.0/gems/spree_backend-3.0.4/app/views/spree/admin/payments/_form.html.erb where line #22 raised:

Missing partial spree/admin/payments/source_forms/_komoju_credit_card with {:locale=>[:en], :formats=>[:html], :variants=>[], :handlers=>[:erb, :builder, :raw, :ruby, :coffee, :jbuilder, :rabl], :versions=>[:v1]}. Searched in:
  * "/Users/adam/code/tacchi/my_rails_app/app/views"
  * "/Users/adam/.rbenv/versions/2.2.0/lib/ruby/gems/2.2.0/gems/spree_komoju-0.0.5/app/views"
  * "/Users/adam/.rbenv/versions/2.2.0/lib/ruby/gems/2.2.0/bundler/gems/spree_address_book-64fd9832859f/app/views"
  * "/Users/adam/.rbenv/versions/2.2.0/lib/ruby/gems/2.2.0/bundler/gems/spree_i18n-a9b909a734cb/app/views"
  * "/Users/adam/.rbenv/versions/2.2.0/lib/ruby/gems/2.2.0/bundler/gems/spree_auth_devise-7b24ad162393/lib/views/backend"
  * "/Users/adam/.rbenv/versions/2.2.0/lib/ruby/gems/2.2.0/bundler/gems/spree_auth_devise-7b24ad162393/lib/views/frontend"
  * "/Users/adam/.rbenv/versions/2.2.0/lib/ruby/gems/2.2.0/gems/devise-3.4.1/app/views"
  * "/Users/adam/.rbenv/versions/2.2.0/lib/ruby/gems/2.2.0/bundler/gems/spree_gateway-7301dc7c84c0/lib/views/backend"
  * "/Users/adam/.rbenv/versions/2.2.0/lib/ruby/gems/2.2.0/bundler/gems/spree_gateway-7301dc7c84c0/lib/views/frontend"
  * "/Users/adam/.rbenv/versions/2.2.0/lib/ruby/gems/2.2.0/gems/spree_frontend-3.0.4/app/views"
  * "/Users/adam/.rbenv/versions/2.2.0/lib/ruby/gems/2.2.0/gems/spree_backend-3.0.4/app/views"
  * "/Users/adam/.rbenv/versions/2.2.0/lib/ruby/gems/2.2.0/gems/spree_api-3.0.4/app/views"
  * "/Users/adam/.rbenv/versions/2.2.0/lib/ruby/gems/2.2.0/gems/spree_core-3.0.4/app/views"
  * "/Users/adam/.rbenv/versions/2.2.0/lib/ruby/gems/2.2.0/gems/kaminari-0.16.3/app/views"```

**Comment by rramsden** [comment]
Hi @robodisco thanks for reporting the issue.

We're currently working on the problem in this pull request: https://github.com/komoju/spree_komoju/pull/32

I'll be sure to ping you once we merge it :-)

**Comment by robodisco** [comment]
awesome thanks @rramsden

**Comment by shioyama** [comment]
This should be fixed now in 0.0.6 (just released). I'll close this, re-open it if you still have issues.
